app/views/hyrax/contact_form/new.html.erb

```diff
@@ -18,35 +18,36 @@
 <% end %>
 
 <%= form_for @contact_form, url: hyrax.contact_form_index_path,
-                            html: { class: 'form-horizontal' } do |f| %>
+                            html: { class: '' } do |f| %>
```

Comment:
I would imagine you don't need to pass any `html:` keyword in this case.  But it does keep the diff cleaner.